src/libopensc/sc-ossl-compat.h

```diff
@@ -50,10 +50,12 @@ extern "C" {
 #if LIBRESSL_VERSION_NUMBER < 0x30500000L
 #define FIPS_mode()                             (0)
 #endif
+#ifndef EVP_sha3_224
 #define EVP_sha3_224()                          (NULL)
 #define EVP_sha3_256()                          (NULL)
 #define EVP_sha3_384()                          (NULL)
 #define EVP_sha3_512()                          (NULL)
+#endif
 #if LIBRESSL_VERSION_NUMBER < 0x3070000fL
```

Comment:
These could probably be
```
#ifndef EVP_PKEY_new_raw_public_key
...
#endif /* ! EVP_PKEY_new_raw_public_key */
```
as a better test for [EVP_PKEY_NEW(3)](https://man.openbsd.org/EVP_PKEY_new_raw_public_key.3).